dev/docker-compose.yaml

```diff
@@ -168,6 +171,20 @@ services:
   #     - ${DOCKER_VOLUME_DIRECTORY:-.}/volumes/etcd:/etcd
   #   command: etcd -advertise-client-urls=http://127.0.0.1:2379 -listen-client-urls http://0.0.0.0:2379 --data-dir /etcd
   #   restart: unless-stopped
+  # intercom:
```

Comment:
I'm not sure that we need it to local build

Reply:
Yes, will be removed and updated Readme with instruction how to use it for testing.